src/Components/SubmissionForm/SubmissionForm.js

```diff
@@ -80,11 +81,30 @@ class SubmissionForm extends Component {
         this.setState({ image: null, image_text: '', error: null });
         setAlert(null);
       })
-      .catch(e => {
-        this.setState({
-          error: e.error
+        .then(res => {
+          //Remove loading spinner
+          this.setState({
+            loading: false,
+            uploading: false,
+          });
+          if (res.status === 400) {
+            setAlert('Sorry, that content is not permitted');
```

Comment:
doesn't the .then block in the api-service check for this (res.ok checks that the status code is within the 200 range)? If it's caught before it gets to this line and you still want the alert to run, this may have to be moved to the api-service doc(?)